avi.wad.Organizer.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id" : "avi.wad.Organizer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "organizer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+            "V" : "1"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{   "name" : "libhandy",
+            "buildsystem" : "meson",
+            "config-opts": [
+                "-Dglade_catalog=disabled",
+                "-Dvapi=false",
+                "-Dtests=false",
+                "-Dexamples=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                  	  "url" : "https://source.puri.sm/Librem5/libhandy.git",
```

Comment:
indenting isn't consistent.

Reply:
will fix that

Reply:
okay I'm confused, my indentation seems okay? I checked other app's json files and theirs seem similar (PasswordSafe)